<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1855</width>
    <height>1091</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_providers">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1851</width>
      <height>1041</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="LH">
     <attribute name="title">
      <string>LH</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_LH">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>781</width>
        <height>961</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>12</number>
      </property>
      <column>
       <property name="text">
        <string>등록일</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>주소</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>보증금</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>월세</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>관리비</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>층</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>면적</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>방</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>화장실</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>ALL Money</string>
      </property>
      <item>
       <property name="text">
        <string>ALL Money</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20000</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbBox_Location">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>GUNJA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SADANG</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbBox_grtMoney">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>ALL Date</string>
      </property>
      <item>
       <property name="text">
        <string>ALL Date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Today</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Within last 3 day</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Within last 7 day</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Within last 15 day</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Within last 30 day</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_search_LH">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_webLH">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>50</y>
        <width>1051</width>
        <height>981</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_webSearch">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_url">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ZB">
     <attribute name="title">
      <string>ZB</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_ZB">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>781</width>
        <height>961</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>등록일</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>주소</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>보증금</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>월세</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>관리비</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>층</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>면적</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_search_ZB">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ZB">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>아차산</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_webZB">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>50</y>
        <width>1051</width>
        <height>971</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_webSearch_2">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_url_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DB">
     <attribute name="title">
      <string>DB</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_DB">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>971</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1855</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
